src/components/pick-list-item/pick-list-item.tsx

```diff
@@ -179,14 +179,14 @@ export class PickListItem implements ConditionalSlotComponent, InteractiveCompon
   @Event({ cancelable: true }) calciteListItemRemove: EventEmitter<void>;
 
   /**
-   * Emits when the the component's label, description, value, or metadata properties are modified.
+   * Emits when the component's label, description, value, or metadata properties are modified.
```

Comment:
This, and similar, doc fixes are unrelated to `overlayPositioning` changes. Please submit separately.

Reply:
Woops, find/replace caught us some stragglers. will do